### ISSUE TITLE
Allow multiple query param examples

### ISSUE THREAD

**Issue by addshore** [enzonotario/vitepress-openapi#130]
### What would you like?

Provide and display multiple query param examples for endpoints

### Why is this needed?

OAPIspec allows it, but only the first is displayed by this component

### How could it be implemented?

Rather than display one line with one value
![image](https://github.com/user-attachments/assets/6cffc123-a57c-47d9-b591-fe0033100dc8)
Multiple lines could be displayed.

**Comment by enzonotario** [comment]
Hi @addshore , I'm working on it in #133, can you please validate it? You can also see a [live example](https://vitepress-openapi-git-feature-par-db760c-enzo-notarios-projects.vercel.app/example/operations/getAllArtists.html)

Thanks!

**Comment by addshore** [comment]
Looks pretty good, but might not be idea currently when the examples are rather long.

![image](https://github.com/user-attachments/assets/fe0cdedb-600e-4bac-b4a8-5b6c1022776b)

In this case, it might be best for them to go onto 1 line each!

Perhaps even larger examples might need a different display.

But I'd also be happy with this PR as it is :)

**Comment by enzonotario** [comment]
Nice! I've made it configurable via

```ts
useTheme({
  spec: {
    wrapExamples: false, // Set false to show them in columns.
  },
})
```

And released in [v0.0.3-alpha.51](https://github.com/enzonotario/vitepress-openapi/releases/tag/v0.0.3-alpha.51)

Thanks!

**Comment by addshore** [comment]
Many thanks, the latest release works well for multiple examples, however I was not able to get `wrapExamples` to work

My code is..

```
export default {
    ...DefaultTheme, Layout,
    enhanceApp({ app, router, siteData }) {
        app.use(vuetify)

        enhanceAppWithTabs(app)

        // Setup Theme
        const themeConfig = useTheme({
            spec: {
              wrapExamples: false, // Set false to show them in columns.
            },
          })
        themeConfig.setLocale('en') // en or es
        themeConfig.setResponseCodeSelector('select')
        theme.enhanceApp({ app })
    }
} satisfies Theme
```
